resources/spec.yaml

```diff
@@ -4655,7 +4655,7 @@ paths:
         - api_token: []
       summary: Get org contact types
       tags:
-        - Org
+        - OrgContactTypeObj
```

Comment:
Could you please explain why we need to change it?

Reply:
It is needed for the golang generator